src/distributors/base-installer.ts

```diff
@@ -0,0 +1,118 @@
+import * as tc from '@actions/tool-cache';
+import * as core from '@actions/core';
+import semver from 'semver';
+import path from 'path';
+import * as httpm from '@actions/http-client';
```

Comment:
why http**m**?

Reply:
Frankly speaking, I can't tell why `httpm`, but we use such notation in 
 - [setup-java](https://github.com/actions/setup-java/blob/d202f5dbf7256730fb690ec59f6381650114feb2/src/installer.ts#L4).
 - [setup-python](https://github.com/actions/setup-python/blob/8c5ea631b2b2d5d8840cf4a2b183a8a0edc1e40d/src/install-pypy.ts#L5)
 - [setup-go](https://github.com/actions/setup-go/blob/37335c7bb261b353407cff977110895fa0b4f7d8/src/installer.ts#L5)

Reply:
fair enough